tests/verification/test_github.py

```diff
@@ -110,6 +92,9 @@ def test_create_repo(enable_network, github_api, testing_repo):
     with pytest.raises(RepoAlreadyExists):
         github_api.create_repo(*args)
 
+    # clean up after the test
+    github_api.delete_repo(*args)
```

Comment:
I think this change is an improvement overall. It's much easier to see where the repo is being created (it's easy to miss it in a fixture). Just one minor thing, which is if there's an error after the initial repo creation, but before the call to delete the repo, then the repo will never get deleted. Wrapping it in a `try..finally` would stop this, it's a bit of a pain but would reduce the chances of having ~2000 repos again.